packages/terra-clinical-no-data-view/src/NoDataView.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import IconNoData from 'terra-icon/lib/icon/IconGlasses';
```

Comment:
How about the glyph we used in our old view model version, can we carry that forward?